src/Feature/SCM/Feature_SCM.rei

```diff
@@ -0,0 +1,64 @@
+[@deriving show]
+type t = {providers: list(Oni_Core.SCMModels.Provider.t)};
```

Comment:
This shouldn't be exposed, but is currently needed to prevent `git.refresh` with not repositories, which should go away soon enough I think.